UI-Session-7-Assignment/javascript.js

```diff
@@ -0,0 +1,339 @@
+var currentForm, currentField,
```

Comment:
1. Check the vehicle number format, It is not working properly.

Please fix this on priority because of this I Got stuck in the middle of the assignment.

Reply:
Chrome has some version issues.
But working fine in Firefox.